### PR TITLE
Fix Tests Makefile ports

### PR DESCRIPTION
**Pull request by rldhont**
The variable `LIZMAP_PORT`, `OWS_PORT` and `WPS_PORT` contained `127.0.0.1`. This commit removed it.

### CODE REVIEW

File: tests/Makefile [hunk 1]
```diff
@@ -14,9 +14,9 @@ QGIS_VERSION_TAG:=3.16
 QGIS_MAP_WORKERS:=1
 WPS_NUM_WORKERS:=1
 
-LIZMAP_PORT:=127.0.0.1:9090
```
Comment (dmarteau) [inline tests/Makefile:17]: No, interface must be explicit for security reason. If user want to change this he may create a docker-compose with its own options.

**Review by dmarteau — COMMENTED** [review]
PR not accepted (see comment)

File: tests/Makefile (hunk 1 above)
Comment (rldhont) [reply tests/Makefile:17]: The error message is not an issue ?

Comment (dmarteau) [reply tests/Makefile:17]: The error message come from a bad local configuration.

**Comment by rldhont** [comment]
If my changes do nothing, @dmarteau you can close my PR

**Comment by nboisteault** [comment]
@rldhont so no need for review?

**Comment by nboisteault** [comment]
@dmarteau @rldhont should this PR be closed?

**Comment by dmarteau** [comment]
Yes AMHA.

**Pull request closed without merging**